vue-js/src/misc/Flash.ts

```diff
@@ -0,0 +1,21 @@
+export class Flash {
```

Comment:
ScanbotSDK has had `this.sdk?.utils.flash();` available for some time. It doesn't exist in all examples because it wasn't part of the SDK initially, but should be used now